bazaarvoice/astyanax
# Astyanax_Jackson_Core_Update_2.10.2

Jira Issue
https://bazaarvoice.atlassian.net/browse/PD-211832

What Are We Doing Here?
SNYK Vulnerability Remediation for Astyanax Jackson Library.
--> Update Jackson fasterxml core (core, annotations, databind) to 2.10.2.

How to Test and Verify

Check out local build :
Unit testing : mvn clean install

![image](https://github.com/bazaarvoice/astyanax/assets/134580247/577f82b8-26de-4793-b437-531035b0a283)


Risk
Level : Low

Required Testing
Manual.

Risk Summary
Since risk level will be low,we will be monitoring for vulnerability issues of jackson library in snyk dashboard.
